client/scss/components/_page_description.scss

```diff
@@ -31,7 +35,7 @@
 }
 
 .page-description__row {
-  margin-bottom: 0.6em;
+  margin-bottom: 0.3em;
```

Comment:
Wherever possible, I think it's good practice to keep to multiples of the `$vertical-space-unit` for horizontal spacing, but in practice – when dealing with type – line-height is applied above and below the line (in contrast to print-design's leading), which throws things off.

I tend to favour the approach that if you can get within a handful of pixels using `$vertical-space-unit` then do that, but if you're stuck awkwardly e.g. 8px away from what the designs call for, then to resort to a different unit.